src/briefcase/commands/build.py

```diff
@@ -132,15 +131,12 @@ def __call__(
         # and that the app configuration is finalized.
         self.finalize(app)
 
-        if apps:
-            selected_apps = {}
-            for name in apps:
-                if name not in self.apps:
-                    raise BriefcaseCommandError(
-                        f"App '{name}' does not exist in this project."
-                    )
-                selected_apps[name] = self.apps[name]
-            apps_to_build = selected_apps
+        if app_name:
+            if app_name not in self.apps:
+                raise BriefcaseCommandError(
+                    f"App '{app_name}' does not exist in this project."
+                )
+            apps_to_build = {app_name: self.apps[app_name]}
```

Comment:
This works - but "good Python" code works on the principle of "it's better to ask forgiveness than permission" - that is - assume the operation will succeed, and catch the error:
```suggestion
            try
                apps_to_build = {app_name: self.apps[app_name]}     
            except KeyError:
                raise BriefcaseCommandError(
                    f"App '{app_name}' does not exist in this project."
                )
```

This ends up being slightly faster, because the "normal" case does the operation, and doesn't need the "pre-check" that the app name is valid.